### PR TITLE
Functional tests - Delete commented tests & fix failed tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Delete commented tests & fix failed tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4025885106
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   | no

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts [hunk 1]
```diff
@@ -351,6 +351,7 @@ describe('BO - Orders - View and edit order: Check invoice', async () => {
     describe('Check the invoice', async () => {
       // Check: Header, Delivery address, Billing address, Invoice number, Invoice date, Order reference and date
       describe('Check Header', async () => {
+        // https://github.com/PrestaShop/PrestaShop/issues/22581
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts]:
```suggestion
        // @todo : https://github.com/PrestaShop/PrestaShop/issues/22581
```

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts [hunk 2]
```diff
@@ -584,6 +585,7 @@ describe('BO - Orders - View and edit order: Check invoice', async () => {
...
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts]:
```suggestion
        // @todo : https://github.com/PrestaShop/PrestaShop/issues/22581
```

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts [hunk 3]
```diff
@@ -779,6 +781,7 @@ describe('BO - Orders - View and edit order: Check invoice', async () => {
...
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts]:
```suggestion
        // @todo : https://github.com/PrestaShop/PrestaShop/issues/22581
```

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts [hunk 4]
```diff
@@ -1019,6 +1022,7 @@ describe('BO - Orders - View and edit order: Check invoice', async () => {
...
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts]:
```suggestion
        // @todo : https://github.com/PrestaShop/PrestaShop/issues/22581
```

File: tests/UI/campaigns/functional/FO/03_userAccount/04_getGDPRDataInPDF.ts [hunk 5]
```diff
@@ -222,6 +227,7 @@ describe('FO - Account : Get GDPR data in PDF', async () => {
     });
 
     describe('Check GDPR data in PDF', async () => {
+      // https://github.com/PrestaShop/PrestaShop/issues/22581
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/03_userAccount/04_getGDPRDataInPDF.ts]:
```suggestion
      // @todo : https://github.com/PrestaShop/PrestaShop/issues/22581
```

**Review by Progi1984 — COMMENTED** [review]
Just added some improvements to comments with link 

=> I transform it in `@todo`. That permits to easily search them in code.

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/11_checkInvoice.ts]: done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**